llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -79,13 +80,13 @@ def __init__(self, config: MPTConfig):
         # both report this helping with stabilizing training
         self.embedding_fraction = config.embedding_fraction
 
-        self.wte = nn.Embedding(config.vocab_size,
+        self.wte = SharedEmbedding(config.vocab_size,
                                 config.d_model,
                                 device=config.init_device)
         if not self.alibi:
-            self.wpe = nn.Embedding(config.max_seq_len,
-                                    config.d_model,
-                                    device=config.init_device)
+            self.wpe = SharedEmbedding(config.max_seq_len,
```

Comment:
I don't think there's any need for `wpe` to be a `SharedEmbedding`. It shouldn't really matter but I think it helps clarify the role of `SharedEmbedding` to limit it to the embedding that is actually shared.